src/lender/adapters/mkr/clerk.sol

```diff
@@ -238,4 +245,16 @@ contract Clerk is Auth, Math {
     function calcOvercollAmount(uint amountDAI) public returns (uint) {
         return rmul(amountDAI, mat());
     }
+
+        // In case contract received DAI as a leftover from the cdp liquidation return back to reserve
+    function returnDAI() public auth {
+        uint amountDAI = dai.balanceOf(address(this));
+        dai.approve(address(reserve), amountDAI);
+        reserve.deposit(amountDAI);
+    }
+
+    function changeOwnerMgr(address usr) public auth {
+        mgr.rely(usr);
+        mgr.deny(address(this));
+    }
```

Comment:
I think thats wrong.
The clerk is the owner and not a `ward`

See: https://github.com/centrifuge/tinlake-maker-lib/blob/master/src/mgr.sol#L129